buildSrc/build.gradle

```diff
@@ -235,6 +209,23 @@ if (project != rootProject) {
     }
   }
 
+  task integTest(type: Test) {
+    inputs.dir(file("src/testKit"))
```

Comment:
We should add a property name (so this doesn't show in scan comparison as $1 or similar) and explicit path sensitivity so we get good cache reuse. We can also directly pass a path to `dir()` no need to convert to a file.

```
inputs.dir("src/testKit").withPropertyName("testKitDir").withPathSensitivity(PathSensitivity.RELATIVE)
```